test/e2e/basic_test.go

```diff
@@ -14,9 +14,33 @@ import (
 	"testing"
 
 	framework "github.com/operator-framework/operator-sdk/pkg/test"
+
+	pravega_e2eutil "github.com/pravega/pravega-operator/pkg/test/e2e/e2eutil"
 )
 
-func testCreateCluster(t *testing.T, f *framework.Framework, ctx *framework.TestCtx) error {
-	t.Skip("Not implemented")
-	return nil
+func testCreateDefaultCluster(t *testing.T) {
```

Comment:
This test seems to be end-to-end. I'm wondering if we should be doing more focused test cases, along the lines of unit tests.

Reply:
@fpj currently there's not a single unit test in the project. I've created issue https://github.com/pravega/pravega-operator/issues/113 to start adding coverage to the project.